test/jdk/javax/swing/JSlider/4987336/bug4987336.java

```diff
@@ -21,25 +21,58 @@
  * questions.
  */
 
-/* @test
-   @bug 4987336
-   @summary JSlider doesn't show label's animated icon.
-   @author Pavel Porvatov
-   @run applet/manual=done bug4987336.html
+/*
+ * @test
+ * @bug 4987336
+ * @summary JSlider doesn't show label's animated icon.
+ * @library /java/awt/regtesthelpers
+ * @build PassFailJFrame
+ * @run main/manual bug4987336
 */
-
-import javax.swing.*;
-import javax.swing.border.TitledBorder;
 import java.awt.event.ActionEvent;
 import java.awt.event.ActionListener;
+import javax.swing.ButtonGroup;
+import javax.swing.BoxLayout;
+import javax.swing.ImageIcon;
+import javax.swing.JComponent;
+import javax.swing.JFrame;
+import javax.swing.JLabel;
+import javax.swing.JPanel;
+import javax.swing.JRadioButton;
+import javax.swing.JScrollPane;
+import javax.swing.JSlider;
+import javax.swing.SwingConstants;
+import javax.swing.SwingUtilities;
+import javax.swing.border.TitledBorder;
+import javax.swing.UIManager;
```

Comment:
`import javax.swing.UIManager;` can be put before `import javax.swing.border.TitledBorder;`